core/templates/components/state-directives/answer-group-editor/answer-group-editor.component.html

```diff
@@ -136,4 +134,10 @@
   .oppia-answer-rule-block {
     border-bottom: 1px solid #ccc;
   }
+
+  @media screen and (max-width: 500px) {
+    strong {
+      font-size: 18px;
+    }
```

Comment:
Please make sure that this style is scoped only to this component.

Reply:
Ditto elsewhere

Reply:
These styles will be component specific (unless they are defined oppia.css)

Reply:
No, they will not, we had issues with this quite recently, make sure that all styles are scoped.

Reply:
Done, used a separate class name.